apps/workers/src/server.ts

```diff
@@ -11,12 +9,7 @@ console.log('Workers started')
 
 const port = process.env.WORKERS_PORT || 3002
 const server = http.createServer(async (req, res) => {
-  const websockets = await WebsocketClient.getSocket()
-
-  if (req.url === '/ping' && req.method === 'GET') {
-    websockets.emit('pingFromWorkers')
-    res.end(JSON.stringify({ status: 'OK', message: 'Pong' }))
-  } else if (req.url === '/health' && req.method === 'GET') {
```

Comment:
remove any other shenanigans that you wrote during the websockets debugging session, if there are,  please

Reply:
I think is clean now, but will check again